<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preferencesWindow</class>
 <widget class="QDialog" name="preferencesWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>270</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>270</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QCheckBox" name="myCheckBoxStartupDialog">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Show the welcome dialog on Startup</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="myCheckBoxBestFit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Best fit when tile or cascade</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="myCheckBoxRecentFiles">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Remember recent files</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="myPushButtonClearRecentFiles">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Crear Recent Files List</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/iconshumanity48/edit-clear.svg</normaloff>:/icons/iconshumanity48/edit-clear.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+R</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>myCheckBoxStartupDialog</zorder>
     <zorder>myCheckBoxBestFit</zorder>
     <zorder></zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Appearance</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Toolbar icon size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="myComboBoxToolbarIconSize">
          <item>
           <property name="text">
            <string>24x24 px</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>32x32 px</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>48x48 px</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="myButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>myCheckBoxRecentFiles</tabstop>
  <tabstop>myPushButtonClearRecentFiles</tabstop>
  <tabstop>myButtonBox</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
